### PR TITLE
Update docs metadata json files with docs config jsons.

### PR DESCRIPTION
**Pull request by sima-zhu**

### CODE REVIEW

File: metadata/latest/Azure.AI.FormRecognizer.json [hunk 1]
```diff
@@ -17,5 +17,8 @@
     "Azure.AI.FormRecognizer.Models",
     "Azure.AI.FormRecognizer.Training",
     "Microsoft.Extensions.Azure"
-  ]
+  ],
+  "DocsConfig": {
+    "Properties": "tfm=netstandard2.0;isPrerelease=true"
```
Comment (weshaggard) [inline metadata/latest/Azure.AI.FormRecognizer.json:22]: I would suggest making this is a direct property instead of nesting.

Comment (sima-zhu) [reply metadata/latest/Azure.AI.FormRecognizer.json:22]: Sure. Just evaluate the necessities of nested structure. I can't think of any benefits beside of good looking. :)

File: metadata/latest/Azure.AI.FormRecognizer.json [hunk 2]
```diff
@@ -17,5 +17,6 @@
     "Azure.AI.FormRecognizer.Models",
     "Azure.AI.FormRecognizer.Training",
     "Microsoft.Extensions.Azure"
-  ]
+  ],
+  "Properties": "tfm=netstandard2.0;isPrerelease=true"
```
Comment (weshaggard) [inline metadata/latest/Azure.AI.FormRecognizer.json:21]:
```suggestion
  "DocPackageProperties": "tfm=netstandard2.0;isPrerelease=true"
```

We should give this a less generic name, so it is clear what it is for. I made one suggestion, but you can name it what makes sense, but it should be something more specific than Properties.

Comment (sima-zhu) [reply metadata/latest/Azure.AI.FormRecognizer.json:21]: Yes, we can avoid name conflicts by having specific name. I would rename to `DocsCiConfigProperties`.

File: metadata/latest/Azure.Batch.FileStaging.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (weshaggard) [inline metadata/latest/Azure.Batch.FileStaging.json:1]: For these newly added files do they need more properties?

**Review by weshaggard — APPROVED** [review]
Left one comment but otherwise aligns with what we talked about.

**Review by danieljurek — APPROVED** [review]
This looks good to me. 

Things we discussed: 
* Will the script which generated these changes need to be run again if new packages release _before_ we merge the metadata changes which write `DocsCiConfigProperties` into these metadata files? -- Yes for completeness, but our onboarding logic has safe defaults which are used in the vast majority of cases.
 
* Do we need to fill in values for `DocsCiConfigProperties` if they're the same as the defaults? -- Not really, but for completeness it's better to see these values instead of looking up where they might come from in onboarding scripts.

* Thinking about the future - In the future we may change how versioning happens, how would these files be affected?

File: metadata/latest/Azure.Batch.FileStaging.json (hunk 3 above)
Comment (sima-zhu) [reply metadata/latest/Azure.Batch.FileStaging.json:1]:
```
azurebatchfilestaging,Azure.Batch.FileStaging
```
This is the only place we defined the package in docs metadata and `azure-dotnet.csv`. I didn't find any other info to fill in.

File: metadata/latest/Azure.AI.FormRecognizer.json [hunk 4]
```diff
@@ -17,5 +17,7 @@
     "Azure.AI.FormRecognizer.Models",
     "Azure.AI.FormRecognizer.Training",
     "Microsoft.Extensions.Azure"
-  ]
+  ],
+  "DocsCiConfigProperties": "tfm=netstandard2.0",
+  "MSDocService": "placeholder"
```
Comment (sima-zhu) [inline metadata/latest/Azure.AI.FormRecognizer.json:22]: Without this value, the package will be filtered out by the 'New' to `true` filter.

**Pull request merged**